plugins/catalog-backend/api-report.md

```diff
@@ -167,6 +167,11 @@ export class CatalogBuilder {
   useLegacySingleProcessorValidation(): this;
 }
 
+// @public (undocumented)
+export type CatalogCollatorEntityTransformer = (
+  entity: Entity,
```

Comment:
Do you see the possibility of having side effects within this role? Is it recommended to return a promise? It seems like returning a promise would conflict with the decorator's concept, but I want to double check with you all.

Reply:
Yeah, indeed, allowing this to return a promise would make it easy/possible to do things that are better handled in decorators.